### DISCUSSION TITLE
highlight.js 添加move语法高亮支持 1000U+500积分，已选择第一个候选人

### DISCUSSION THREAD

**Discussion by uvd** [starcoinorg/web3startrek#113] [MoveProjects]
https://github.com/highlightjs/highlight.js 

前面的工程师其实已经实现了绝大部分功能了的
https://github.com/MystenLabs/sui/tree/main/doc/book 参考这本书的实现方式

验证条件
1. sui-framework 代码高亮能正常 https://github.com/MystenLabs/sui/tree/main/crates/sui-framework/packages 
2. aptos-framework 代码高亮能正常 https://github.com/aptos-labs/aptos-core
3. starcoin-framework 代码高亮能正常     https://github.com/starcoinorg/starcoin-framework
4. 非必须 尝试修正 https://move-book.com/cn  和 https://github.com/move-language/move/tree/main/language/documentation/book 生成的页面的语法高亮

发放时间节点，测试大概没问题，代码合并进去 https://github.com/highlightjs/highlight.js  一次性支付800U
期间要是遇到bug或者什么的需要修复一下，一个月以后支付剩下的 200U

理想的候选人，能持续修复一下语法上支持的小bug

**Comment by zven21** [comment]
🙋🏻‍♀️

**Comment by zar1111** [comment]
认领

**Comment by chrisyy2003** [comment]
认领！！！👍👍👍

**Comment by uvd** [comment]
@chrisyy2003 @zar1111  已经选中第一个候选人 O_O

**Comment by peter209393** [comment]
目前进展：
https://github.com/damirka/move-book/pull/27
https://github.com/damirka/move-book/pull/28
https://github.com/move-language/move/pull/1049
https://github.com/MystenLabs/awesome-move/pull/84
